src/cmd/linuxkit/run_qemu.go

```diff
@@ -289,6 +303,7 @@ func runQemu(args []string) {
 		Memory:         *mem,
 		KVM:            *enableKVM,
 		Containerized:  *qemuContainerized,
+		QemuOptions:    strings.Split(*qemuDirectOptions, " "),
```

Comment:
This won't allow any QEMU options which happen to contain a space. Maybe that is not an issue in practice? An alternative would be to allow the option to be given multiple times to populate a slice.